bundle/manifests/apps.3scale.net_apicasts.yaml

```diff
@@ -38,13 +38,14 @@ spec:
             description: APIcastSpec defines the desired state of APIcast
             properties:
               adminPortalCredentialsRef:
-                description: LocalObjectReference contains enough information to let you locate the referenced object inside the same namespace.
+                description: Secret reference to a Kubernetes Secret containing the portal endpoint URL information. The Secret must be located in the same namespace
```

Comment:
```suggestion
                description: Secret reference to a Kubernetes Secret containing the admin portal endpoint URL. The Secret must be located in the same namespace.
```

Reply:
Updated :+1: .
I also added periods at the end of all descriptions